### ISSUE TITLE
nbopen executable not installed under windows

### ISSUE THREAD

**Issue by tritemio** [takluyver/nbopen#18]
1. I first installed nbopen with pip or with distutils.

2. Then I ran `python win-install.py` from the source tree

3. `nbopen` is not found:

```
'nbopen' is not recognized as an internal or external command,
operable program or batch file.
```

Windows 7, Anconda 2.2 64, python 2.7

**Comment by takluyver** [comment]
Yeah, on Windows it doesn't set up a command line shortcut. Double click should work in Windows explorer, and you can run `python -m nbopen foo.ipynb` at the command line.

**Comment by tritemio** [comment]
Oh yes, you are right, initially the double-click was silently failing due to the out-of-home-dir issue.

**Comment by ajasja** [comment]
Just wanted to add that when using git for windows (so basically bash on windows) nbopen works great from the command line as well.
